cmd/image-builder/logging.go

```diff
@@ -42,24 +44,33 @@ func (h *ctxHook) Fire(e *logrus.Entry) error {
 // Extract/generate request id and store it in the standard context
 func requestIdExtractMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
 	return func(c echo.Context) error {
-		// extract or generate request id
+		// extract or generate request and insights id
 		rid := c.Request().Header.Get("X-Rh-Edge-Request-Id")
```

Comment:
Why do we prefer Edge here, would Insights make sense here, to have just single request id?

Reply:
Yeah but as I explain in the description, edge-id was recommended to me on Slack, however, the problem is it is not propagated by Edge API to the client via HTTP header, only insights-id is. So I propose to use both, so it is super easy to find records if you either have one, or the other.

Reply:
okey, sure :+1: